oat-sa/tao-item-runner-qti-fe
# feat/TR-5016/convert-text-responses

related to https://oat-sa.atlassian.net/browse/TR-5016

### Description

Responses which contain ambiguous symbols need to be transformed to ascii symbols before submitting to backend
This PR reuses core/util converter to do the transformation

### How to test 

beware of this dependency : `"@oat-sa/tao-core-sdk": "https://github.com/oat-sa/tao-core-sdk-fe#feature/TR-5014/text-converter",
`

Run unit tests 
`npm run build && npm run test:keepAlive`



Create an item containing TextEntry and ExtendedText interaction, preview it enter some ambiguous chars (i.e  `  １２ `) and submit the response, check the payload

For TextEntry interaction it should trim and convert ambiguous chars to ascii ones
For ExtendedText interaction it should just convert ambiguous chars to ascii ones
